### PR TITLE
ref(icons): Rotate & rename IconSwitch as IconSort

### PR DESCRIPTION
**Pull request by vuluongj20**
Replace `IconSwitch` with a rotated version, named `IconSort`.

**Before:**
Screen Shot 2022-03-01 at 11 41 18 AM

**After:**
Screen Shot 2022-03-01 at 11 40 43 AM

### CODE REVIEW

File: static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx [hunk 1]
```diff
@@ -48,4 +50,8 @@ function Icon({type}: Props) {
   }
 }
 
+const IconHttp = styled(IconSort)`
+  transform: rotate(90deg);
+`;
+
```
Comment (vuluongj20) [inline static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx:56]: Screen Shot 2022-03-01 at 11 35 04 AM

File: static/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx [hunk 2]
```diff
@@ -233,7 +233,8 @@ const Time = styled('div')`
   cursor: pointer;
 `;
 
-const StyledIconSwitch = styled(IconSwitch)`
+const IconSwitch = styled(IconSort)`
+  transform: rotate(90deg);
```
Comment (vuluongj20) [inline static/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx:237]: Screen Shot 2022-03-01 at 11 33 38 AM

File: static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx [hunk 3]
```diff
@@ -48,4 +50,8 @@ function Icon({type}: Props) {
   }
 }
 
+const IconHttp = styled(IconSort)`
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx:53]: Should the icon just have a rotate prop?

Comment (vuluongj20) [reply static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx]: hmm good point! Thought of these rotated versions as edge cases (in the future we want to have a separate, better icon for `http`), so didn't think of adding a custom prop for them, but until then a prop is more neat. Added in https://github.com/getsentry/sentry/pull/32176/commits/e3d7cae9fac12b197b7fe1309d0e25443c3a57e8.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**